third_party/nvfuser/csrc/python_frontend/python_bindings.cpp

```diff
@@ -2586,6 +2586,72 @@ void initNvFuserPythonBindings(PyObject* module) {
       py::arg("correction"),
       py::arg("keepdim") = false,
       py::return_value_policy::reference);
+  nvf_ops.def(
+      "uniform",
+      [](FusionDefinition::Operators& self,
+         Scalar minval,
```

Comment:
Was this supposed to have python numbers as min and max values as well for the random factory functions?